File: src/server/gkill/api/req_res/add_kmemo_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type AddKmemoResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	AddedKmemo *reps.Kmemo `json:"added_kmemo"`

	AddedKyou *reps.Kyou `json:"added_kyou"`
}

File: src/client/classes/api/req_res/add-kmemo-response.ts
'use strict'

import { Kmemo } from '@/classes/datas/kmemo'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class AddKmemoResponse extends GkillAPIResponse {

    added_kmemo: Kmemo

    added_kyou: Kyou | null

    constructor() {
        super()
        this.added_kmemo = new Kmemo()
        this.added_kyou = null
    }

}


